recipes-ros/mavros/mavros.inc

```diff
@@ -0,0 +1,15 @@
+#
+# MAVROS -- MAVLink extendable communication node for ROS with proxy for Ground Control Station.
+#
+
+SRCBRANCH = "indigo-devel"
+SRC_URI = "git://github.com/mavlink/mavros.git;protocol=http;branch=${SRCBRANCH}"
```

Comment:
Are the mavros sources available as released archive file? The git repository should only be used if there is no archive available.